README.md

```diff
@@ -95,6 +112,25 @@ The returned event payload is the same as the input.
         "message": "..."
     }
 
+### ReactionAdded
+    {
```

Comment:
This seems to be wrong according to
```
type reactionAddedEvent struct {
	ReactionUser   user   `json:"user"`
	ReactionName   string `json:"reaction"`
	EventTimestamp string `json:"eventTimestamp"`
	ItemType       string `json:"type"`
	ItemTimestamp  string `json:"itemTimestamp"`
	ItemUser       user   `json:"itemUser"`
	ChannelId      string `json:"channelId"`
}
```
There is no such thing as `item` and users are not simple string but objects/structs in itself

Reply:
There is some reason, which version are you referring to? slack@v0.9.1>websocket_reactions.go contains below defination


`
// reactionItem is a lighter-weight item than is returned by the reactions list.
type reactionItem struct {
	Type        string `json:"type"`
	Channel     string `json:"channel,omitempty"`
	File        string `json:"file,omitempty"`
	FileComment string `json:"file_comment,omitempty"`
	Timestamp   string `json:"ts,omitempty"`
}

type reactionEvent struct {
	Type           string       `json:"type"`
	User           string       `json:"user"`
	ItemUser       string       `json:"item_user"`
	Item           reactionItem `json:"item"`
	Reaction       string       `json:"reaction"`
	EventTimestamp string       `json:"event_ts"`
}

// ReactionAddedEvent represents the Reaction added event
type ReactionAddedEvent reactionEvent

// ReactionRemovedEvent represents the Reaction removed event
type ReactionRemovedEvent reactionEvent
`